theano/sandbox/cuda/basic_ops.py

```diff
@@ -320,10 +321,18 @@ class GpuDimShuffle(GpuOp):
 
     check_broadcast = False
 
-    def __init__(self, input_broadcastable, new_order):
+    __props__ = ("input_broadcastable", "new_order", "inplace")
```

Comment:
I need to have inplace here because there are some optimizations that uses tensor.DimShuffle 's parameters to GpuDimShuffle. So I have added this parameter, and asserted to be always True